GOVERNANCE.md

```diff
@@ -1,45 +1,117 @@
-# Governance
+> **Note:**
+>
+> This is a **Work in Progress**. Stay tuned for more follow-up updates!
+
+# TiKV Governance
+
+ This document describes the governance rules of the TiKV project (organization). It is meant to be followed by all the repositories in the project and the TiKV project.
+
+- [Code of Conduct](#code-of-conduct)
+- [Community groups and roles](#community-groups-and-roles)
+    - [Maintainer](#maintainer)
+    - [SIGs](#sigs)
+    - [Users](#users)
+- [Approving PRs](#approving-prs)
+- [Decision making and voting](#decision-making-and-voting)
+    - [Proposal process](#proposal-process)
+- [Adding new SIGs/projects](#adding-new-sigsprojects)
 
 ## Code of Conduct
 
-The TiKV community follows the TiKV Code of Conduct. Here are some excerpts:
+The TiKV community follows the [CNCF Code of Conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md).
+
+## Community groups and roles
+
+The TiKV project is comprised of the following types of contributing groups:
+
+- Maintainers
+- Special Interest Groups (SIGs)
+- Users
+
+### Maintainer
+
+Maintainers are first and foremost contributors that have shown they are committed to the long term success of a project. They are the planners and designers of the TiKV project, with the authority to merge branches into the master branch. Maintainership is about building trust with the current maintainers of the project and being a person that they can depend on to make decisions in the best interest of the project in a consistent manner.
+
+While maintainership indicates a valued member of the community who has demonstrated a healthy respect for the project’s aims and objectives, their work must be reviewed by the community before acceptance in an official release.
+
+A Maintainer is not allowed to merge their change without approval from another person. However, they are allowed to sidestep this rule under justifiable circumstances. For example:
+
+- If a CI tool is broken, they may override the tool to still submit the change.
```

Comment:
The change should still be reviewed it is just *merged* manually, so this does not quite fit.